src/main/java/codeu/model/store/basic/DefaultDataStore.java

```diff
@@ -187,4 +187,20 @@ private String getRandomMessageContent() {
 
     return messageContent;
   }
+
+  private String getRandomAboutContent() {
+    String loremIpsum =
+        "dolorem ipsum, quia dolor sit amet consectetur adipiscing velit, "
+            + "sed quia non numquam do eius modi tempora incididunt, ut labore et dolore magnam "
+            + "aliquam quaerat voluptatem. Ut enim ad minima veniam, quis nostrum exercitationem ullam "
+            + "corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum "
+            + "iure reprehenderit, qui in ea voluptate velit esse, quam nihil molestiae consequatur, vel illum, "
+            + "qui dolorem eum fugiat, quo voluptas nulla pariatur";
+
+    int startIndex = (int) (Math.random() * (loremIpsum.length() - 100));
+    int endIndex = (int) (startIndex + 10 + Math.random() * 90);
+    String aboutContent = loremIpsum.substring(startIndex, endIndex).trim();
+
+    return aboutContent;
+  }
```

Comment:
Why do we generate a random about content for every user, instead of having a default empty string or a default block of text?

Reply:
I wanted to generate random about content just so every default user wouldn't all have all empty strings or all the same about, kind of mirroring how actual users would be.

Reply:
This might be a little over-engineering with having lots of different random strings. IMHO YAGNI, I would just use a constant, unless you are really trying to show that you have separate users that are decoupled, which I think should be a separate test point anyways from this change.